interpreter/cling/test/Pragmas/add_env_path.C

```diff
@@ -0,0 +1,24 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+// RUN: mkdir -p %T/subdir && clang -shared %S/call_lib.c -o %T/subdir/libtest%shlibext
+// RUN: cd %T/subdir && %S/create_header.sh && cd %S
```

Comment:
What is create_header.sh? Why don't you just write a header and check it into the repository?